scalableminds/webknossos
# Fix project link

### Mailable description of changes:
 - Fix: Clicking on a project within the task list page was fixed so that the project page will actually only show the linked project.

### URL of deployed dev instance (used for testing):
- https://fixprojectlink.webknossos.xyz

### Steps to test:
- click on the project link within the task list view --> project page should be loaded and search value should be filled with the project name
- visit projects page via the navbar (everything should work)
- search for something in the project page and reload the page --> search query should be persisted

### Issues:
- fixes #2756 

------
